core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.ts

```diff
@@ -94,23 +94,15 @@ angular.module('oppia').component('contributorDashboardPage', {
 
       ctrl.onTabClick = function(activeTabName) {
         ctrl.activeTabName = activeTabName;
-        // The $timeout is required because at execution time,
-        // the element may not be present in the DOM yet. Thus, it ensures
-        // that the element is visible before focusing.
+        // The $timeout is required to ensure that focus is applied only
+        // after all the functions in main thread have executed.
         if (ctrl.activeTabName === 'translateTextTab') {
           $timeout(() => {
-            ctrl.addFocusWithoutScroll('selectLangDropDown');
-          }, 0);
+            FocusManagerService.setFocusWithoutScroll('selectLangDropDown');
+          }, 5);
```

Comment:
Why is the timeout set to 5 here?

Reply:
done. Changed it back to 0, it was working with 0.